Sample/swift-sdk-sample/swift-sdk-sample/AppDelegate.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AppDelegate.swift
+//  swift-sdk-sample
+//
+//  Created by Dung Le on 2019/5/14.
+//  Copyright © 2019 Bitmark. All rights reserved.
+//
+
+import UIKit
+import BitmarkSDK
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        
+        SDKInitializationSample.initialize(config: SDKConfig(apiToken: AppConstants.API_TOKEN,
```

Comment:
Best practice here is to set via Info.plist, so we can separate `api_token` by iOS target.